css/src/atomics/border.scss

```diff
@@ -111,3 +111,21 @@
 		border: $border-width solid $border-yellow-high-contrast !important;
 	}
 }
+
+// Gradients
+$border-gradient-themes: 'argon' $info-vivid $primary, 'sunrise' $warning $danger;
+
+@mixin border-gradient($deg, $colorStart, $colorEnd) {
+	border-color: transparent !important;
+	background: linear-gradient(var(--theme-body-background) 0 0) padding-box,
```

Comment:
Should we have a variation for `--theme-body-background-accent` or `--theme-body-background-medium` since they can't be dymanically updated with background classes?

Reply:
Looking over this solution and it has some really nice upsides, but one principal of atomics (and our components too) is that they need to be composable. That is to say, they need to "just work" when layered with other classes that target different properties. That said, this works great with border radius and border-width, but is _so_ nice. 

I've no had much time to mess around with this, but is there any avenue available for making the background of the `padding-box` transparent, rather than applying a specific color? If we could do that (or revamp to use `border-image`) in some way, then we could hit all the required checkpoints.

Reply:
Resolving conversation, `border-image` isn't compatible with `border-radius`